src/scores/categorical/contingency_impl.py

```diff
@@ -102,7 +102,7 @@ def accuracy(self) -> xr.DataArray:
 
         Notes:
 
-            - Range: 0 to 1, where 0 indicates no skill, and 1 indicates a perfect score.
+            - Range: 0 to 1, where 1 indicates a perfect score.
```

Comment:
I wasn't sure it was right to say 0 indicates the model has no skill, in case other metrics could reveal a model has skill despite zero accuracy. Hard to see exactly how this would happen, but I thought I'd keep the claim to what is a perfect score.

Reply:
Your update makes sense. It's probably more likely to be the other way around - i.e., random forecasts would likely lead to non-zero accuracy, but in reality there is no skill. 

You can only get zero accuracy if your only get false positives and/or false negatives, in which case you should flip your event/non event forecasts and get perfect accuracy.